keras/layers/advanced_activations.py

```diff
@@ -102,14 +102,13 @@ class ELU(Layer):
         - [Fast and Accurate Deep Network Learning by Exponential Linear Units (ELUs)](http://arxiv.org/pdf/1511.07289v1.pdf)
     '''
     def __init__(self, alpha=1.0, **kwargs):
+        warnings.warn('ELU layer is deprecated. Use Activation(\'elu\') instead.')
```

Comment:
Not deprecated, since the layer is still needed for specifying alpha and serializing it as part of your model.